app/controllers/users_controller.rb

```diff
@@ -0,0 +1,36 @@
+class UsersController < ApplicationController
+  respond_to :html
+  before_action :authenticate_user!
+  load_and_authorize_resource
+
+  def index
+    @users = User.all.order('lower(email)')
```

Comment:
might be better to write a scope :all_by_email on the User model, and then here do User.all_by_email